flow-typed/overrides/Window.js

```diff
@@ -47,7 +37,7 @@ declare class Window extends EventTarget {
   requestAnimationFrame: typeof requestAnimationFrame,
   devicePixelRatio: number,
   indexedDB: IDBFactory,
-  IDBKeyRange: IDBKeyRange,
+  IDBKeyRange: IDBKeyRange<>,
```

Comment:
Oops: we were using Flow's IDBKeyRange definition here, while we were using our (better) IDBKeyRange implementation elsewhere. This turned Flow crazy... (see https://github.com/facebook/flow/issues/4811)

Here I don't specify a parameter to IDBKeyRange because actually this property isn't an instance of it so I didn't know what to put in it. Now I don't really know how Flow handles this, but it seems happy so far.